src/action/status.rs

```diff
@@ -1,26 +1,231 @@
-use crate::{Workspace, Config};
 use crate::cargo;
 use crate::git;
 use crate::github;
 use crate::package;
+use crate::{Config, Workspace};
 
 use semver::Version;
 
+use slog::*;
+
+/**
+WORKSPACE STRUCTURE
```

Comment:
Maybe include this in some doc file to avoid cluttering the source (it is a bit noisy).